brave/brave-core
# Use interstitial page instead of infobar for ipfs

Resolves https://github.com/brave/brave-browser/issues/13655

- Implemented interstitial page for IPFS ASK mode in settings
- Show IPFS interstitial instead of IPFS Infobar
- Show node installation progress and errors on page
- Removed IPFS Infobar


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Set IPFS mode in settings as ASK
* Navigate any ipfs url, `ipfs://bafkreigcnxudvpojjfwncmauociy5q46zsq46oe66cxbyzie3imabuoege` for example
* You should see the interstitial page with source url in address bar
* In case of error during IPFS node installation(or launching), an error message should be visible
* Change browser theme in settings, the page should apply new theme without reloading
